src/spec-configuration/configuration.ts

```diff
@@ -21,10 +21,17 @@ export type UserEnvProbe = 'none' | 'loginInteractiveShell' | 'interactiveShell'
 
 export type DevContainerConfigCommand = 'initializeCommand' | 'onCreateCommand' | 'updateContentCommand' | 'postCreateCommand' | 'postStartCommand' | 'postAttachCommand';
 
+export interface HostGPURequirements {
```

Comment:
I recently added `mergeHostRequirements()` in `imageMetadata.ts` that needs to be updated to also merge the GPU requirements. (The merge logic is to go with the max values.)

Reply:
I think I've updated - however not 100% sure where that is used, so I'm not very confident about it.

Reply:
I've added to the tests to check the functioning of the `mergeHostRequirements`